packages/expect/src/matchers.js

```diff
@@ -44,6 +45,9 @@ type ContainIterable =
   | DOMTokenList
   | HTMLCollection<any>;
 
+const diff: typeof jestDiff = (a, b, options) =>
```

Comment:
Not sure if there's a better place to put this, `jest-matcher-utils` seems bad because it would then have to depend on `jest-diff`.